pyproject.toml

```
@@ -81,6 +82,27 @@ yt-dlp = "yt_dlp:main"
 [project.entry-points.pyinstaller40]
 hook-dirs = "yt_dlp.__pyinstaller:get_hook_dirs"
 
+[tool.hatch.envs.default]
+dependencies = ["yt-dlp[dev]"]
```

could we do this? what is the difference?
```suggestion
features = ["dev"]
```